Src/FluentAssertions/Specialized/AsyncFunctionAssertions.cs

```diff
@@ -10,11 +10,11 @@ namespace FluentAssertions.Specialized
     /// Contains a number of methods to assert that an asynchronous method yields the expected result.
     /// </summary>
     [DebuggerNonUserCode]
-    public class AsyncFunctionAssertions
+    public abstract class AsyncFunctionAssertions
     {
         private readonly IExtractExceptions extractor;
 
-        public AsyncFunctionAssertions(Func<Task> subject, IExtractExceptions extractor)
+        protected AsyncFunctionAssertions(Func<Task> subject, IExtractExceptions extractor)
```

Comment:
Making constructor protected is a breaking change, isn't it?

Reply:
You're right as well.